app/gameengine.py

```diff
@@ -2758,6 +2783,9 @@ def do_effect(self, effect_player : PlayerState, effect):
                             holomem_targets = [holomem for holomem in holomem_targets if any(color in holomem["colors"] for color in limitation_colors)]
                         case "last_chosen_holomem":
                             holomem_targets = [holomem for holomem in holomem_targets if holomem["game_card_id"] == self.last_chosen_holomem_id]
+                        case "name_in":
```

Comment:
Maybe I'm missing something? This seems unused int he definitions

Reply:
These should've been removed prior to the commit.